src/core_editor/edit_stack.rs

```diff
@@ -57,11 +57,6 @@ where
         self.internal_list = vec![T::default()];
     }
 
-    /// List out all the entries on the undo stack Mostly used for debugging. Might remove this one
-    pub(super) fn edits<'a>(&'a self) -> Box<dyn Iterator<Item = &'a T> + 'a> {
-        Box::new(self.internal_list.iter().take(self.index + 1))
-    }
-
     /// Return the entry currently being pointed to
     pub(super) fn current(&mut self) -> &T {
```

Comment:
I am not sure what the convention is in rust land but my assumption is that this particular module should only be accessible in the editor module, so I added this constraint.

Reply:
Don't have opinions on that. I tend to spam `pub(crate)` for convenience